services/linetracking.md

```diff
@@ -0,0 +1,31 @@
+# Line tracking
+
+A sensor that detects light and dark surfaces, typically used for line following robots.
+
+    identifier: 0x126c4cb2
+    extends: _sensor
+
+## Registers
+
+    ro brightness: u8 / @ reading
+
+Reports the detected brightness. It may be a digital value or, for some sensor, analog value.
+
+    enum Variant: u8 {
+        InfraredDigital = 1,
+        InfraredAnalog = 2,
+        InfraredAnalog5ch = 3
+    }
+    const variant?: Variant @ variant
```

Comment:
again, not sure what is analog/digital here

Reply:
analog output

Reply:
oh, so digital one would report either 0 or 255. got it.

Reply:
the 5ch one - I guess it would tell you where the line is (ie the value at each of the 5 channels), wouldn't it?